client/blocks/editor-checkout-modal/index.tsx

```diff
@@ -71,7 +71,7 @@ const EditorCheckoutModal: React.FunctionComponent< Props > = ( props ) => {
 			onRequestClose={ onClose }
 			title={ String( translate( 'Checkout modal' ) ) }
 			shouldCloseOnClickOutside={ false }
-			icon={ <Icon icon={ wordpress } size={ 36 } /> }
+			icon={ <Icon icon={ wordpress } size={ 29 } /> }
```

Comment:
The `<Icon/>` accepts any `number` for the size attribute but the [docs](https://wordpress.github.io/gutenberg/?path=/docs/components-icon--sizes) a fixed range is proposed. 

Do we want, for the sake of consistency, to use the _defaults_ (e.g. `28` or `32`) or defer from it?